ephios/core/templates/core/shift_copy.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% load static %}
+{% load i18n %}
+
+{% block title %}
+    {% translate "Copy shift" %}
+{% endblock %}
+
+{% block css %}
+{% endblock %}
+
+{% block content %}
+    <div class="page-header">
+        <h1>
+            {% translate "Copy shift" %} "{{ shift.event.title }}" <small>({{ shift.get_datetime_display }})</small>
```

Comment:
I think if a user just wants *duplicate* a shift the current setup does not explain well what is going on. They would first need to figure out how to get a *single* copy then after confirming open the shift edit view again.
Therefore maybe having the default for a single copy at the original time would be good, or a seperate view for duplicating a shift out of the shift edit view that just sends you straight to the edit view of the duplicate.

Reply:
wontfix

Reply:
I don't think this needs a separate View, but just some GET params steering the view to make a single copy. I do agree we should do this if the need actually comes up.

Reply:
I don't think this needs a separate View, but just some GET params steering the view to make a single copy. I do agree we should do this if the need actually comes up.